azerothcore/azerothcore-wotlk
# chore(issue/pr_template): rework and modernize templates

## 🍰 Pull request


- New Issue templates
- New PR template
- Icons for links in the Issue choose menu

New Issue selection + icons for each link on issue/choose:
![image](https://user-images.githubusercontent.com/1238238/109397550-7bac8c80-7937-11eb-84d3-e10c43698632.png)

Furthermore this PR reassembles the new PR template

### Issues Addressed:

- None
### Tests Performed:

- Created some issues on the fork
- Visual Tests regarding the issues
- Labels are created successfully
- PR visual test via preview

### How to Test the Changes:


Go to [my fork](https://github.com/ulfgebhardt/azerothcore-wotlk/issues) and [open an Issue](https://github.com/ulfgebhardt/azerothcore-wotlk/issues/new/choose)

You can see an blank issue opened for each issue type [there](https://github.com/ulfgebhardt/azerothcore-wotlk/issues):

![image](https://user-images.githubusercontent.com/1238238/109397589-bc0c0a80-7937-11eb-9e9a-d53a4d84f460.png)

### Known Issues and TODO List:

- [ ] Labels need to be created if they should be used. Those labels are `bug`, `feature`, `npc`, `spell`, `quest` and `data`. Those are optional but bring the benefit of auto labeling.







---



| When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**. |
| --- |


| You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions [here](http://www.azerothcore.org/wiki/How-to-test-a-PR) |
| --- |
